api/routes.go

```diff
@@ -36,8 +36,13 @@ func (s *server) routes() {
 	api.HandleFunc("/{account}/movers/{group}", s.MoverListHandler).Methods(http.MethodGet)
 	api.HandleFunc("/{account}/movers/{group}/{name}", s.MoverShowHandler).Methods(http.MethodGet)
 	api.HandleFunc("/{account}/movers/{group}/{name}", s.MoverDeleteHandler).Methods(http.MethodDelete)
-	// api.HandleFunc("/{account}/movers/{group}/{name}", s.MoverUpdateHandler).Methods(http.MethodPut)
+	// api.HandleFunc("/{account}/movers/{group}/{name}/{id}", s.MoverUpdateHandler).Methods(http.MethodPut)
+
+	api.HandleFunc("/{account}/movers/{group}/{name}/runs", s.StartTaskHandler).Methods(http.MethodPost)
+	api.HandleFunc("/{account}/movers/{group}/{name}/runs", s.StopTaskHandler).Methods(http.MethodPut)
+	api.HandleFunc("/{account}/movers/{group}/{name}/startStop", s.StartStopTaskHandler).Methods(http.MethodPut)
```

Comment:
I think you can remove these 3 routes and replace them with the one above that's commented out
Let's use the `MoverUpdateHandler` to do the starting and stopping

Reply:
Replaced the routes with MoverUpdateHandler ..